backend/layers/processing/process.py

```diff
@@ -82,6 +81,7 @@ def log_batch_environment(self):
             "MAX_ATTEMPTS",
             "MIGRATE",
             "REMOTE_DEV_PREFIX",
+            "TASK_TOKEN",
```

Comment:
This is not included in the sfn definition yet

Reply:
Good catch.